cfgov/jinja2/v1/_includes/organisms/ask-search.html

```diff
@@ -54,7 +63,23 @@
                                name="q"
                                id="o-search-bar_query"
                                value="{{ ask_query }}"
-                               placeholder="{{ placeholder }}">
+                               placeholder="{{ placeholder }}"
+                               maxlength="{{ max_length }}"
+                               aria-describedby="o-search-bar-error_message">
+                    </div>
+                    <div class="a-form-alert a-form-alert__error u-hidden"
+                         id="o-search-bar-error_message"
```

Comment:
Technically in BEM you'd have a block with an element inside it. In this case we have `o-search-bar-error` as the block and `_message` as the element. We don't have a `o-search-bar-error` ID as a container. We do have `o-search-bar` as a class. We may want to make this `o-search-bar_error-message` to say even though this is an ID it is the `_error-message` element of the `o-search-bar` block.

Reply:
Fixed in 608db2c.